File: results/WizardCoder/javascript_React/sortlines.js
import React, { useState } from'react';

function App() {
  const [inputText, setInputText] = useState('');
  const [outputText, setOutputText] = useState('');

  const handleInputChange = (event) => {
    setInputText(event.target.value);
  };

  const handleProcessClick = () => {
    const lines = inputText.split('\n');
    const uniqueLines = [...new Set(lines)];
    const sortedLines = uniqueLines.sort();
    setOutputText(sortedLines.join('\n'));
  };

  return (
    <div>
      <textarea value={inputText} onChange={handleInputChange} />
      <button onClick={handleProcessClick}>Process</button>
      <textarea value={outputText} readOnly />
    </div>
  );
}

export default App;

File: results/ChatGPT/python_streamlit/sortlines.py
import streamlit as st

def process_text(text):
    lines = text.split('\n')
    sorted_lines = sorted(set(lines))
    return '\n'.join(sorted_lines)

st.title("Text Processor")

input_text = st.text_area("Enter your text here", height=400)
if st.button("Process"):
    output_text = process_text(input_text)
    st.text_area("Processed Text", value=output_text, height=400)